src/io/flutter/editor/InlinePreviewViewController.java

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
This class isn't executed in the checked in code for now as showing previews directly in the code editor is disabled to avoid risking memory leaks or rendering performance issues on slow machines.